readme.md

```diff
@@ -1,17 +1,31 @@
 # Construct
 
+Construct helps you to quickly setup your computer with the tools we have found helpful here at Segment.
 
-Install construct
+Construct will install the following tools:  
 
-  git clone git@github.com:pivotal-cloudops/construct.git
+  atom
```

Comment:
If we are going to do list all the things installed we have a tightly coupled dependency with the readme and with the programs therein.  Documentation is notorious for falling behind it's code.  This seems dangerous.  If i were writing this and expected other people to follow my conventions i would write a script to generate this part of your readme directly from your folder structure.  Then i would run that script when i was ready to commit.  Alternatively you can tell your audience that every folder name will install that thing.  Your call.